hikari/impl/bot.py

```diff
@@ -905,7 +924,7 @@ def _max_concurrency_chunker(self) -> typing.Iterator[typing.Iterator[int]]:
 
     async def _abort_shards(self) -> None:
         """Close all shards and wait for them to terminate."""
-        for shard_id in self._tasks:
+        for shard_id in self._shards:
```

Comment:
Worst we can do is shut down a shard before it starts, which should just stop it from trying to start at all.